### DISCUSSION TITLE
posts/Tech-Blog/

### DISCUSSION THREAD

**Discussion by giscus** [hajeonghun/hajeonghun.github.io#3] [General]
# posts/Tech-Blog/

기술 관련 포스팅을 진행할 예정입니다. 습득한 지식들을 제대로 기록하지 않아, 시간이 지나 잊게되는 모습을 보며 기록의 필요성을 느꼈습니다. 노션(Notion) 혹은 수기로 여기저기 정리하다보니, 한 곳에서 관리할 필요성을 느꼈습니다. 추후 나를 위한 기록이지만, 타인에게 설명한다고 생각하면 더 깊이있게 공부할 수 있는 걸 느꼈습니다.

https://hajeonghun.github.io/posts/Tech-Blog/

**Comment by hajeonghun** [comment]
댓글 테스트
